exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -96,6 +96,27 @@ fun <T : Any?> ExpressionWithColumnType<T>.varPop(scale: Int = 2): VarPop<T> = V
  */
 fun <T : Any?> ExpressionWithColumnType<T>.varSamp(scale: Int = 2): VarSamp<T> = VarSamp(this, scale)
 
+// using `Op`
+
+/** Returns this array of data wrapped in the `ALL` operator. */
+fun <T> Array<T>.allOp(): Op<T> = AllAnyOp("ALL", this)
+
+/** Returns this array of data wrapped in the `ANY` operator. The name is explicitly distinguished from [Array.any]. */
+fun <T> Array<T>.anyOp(): Op<T> = AllAnyOp("ANY", this)
```

Comment:
Having these be extension functions makes it difficult to extend the functionality to, for example, a subquery. It also doesn't read like a natural SQL query: `WHERE { Users.id eq arrayOf(1, 2).anyOp() }`.

Please replace them with `anyFrom` which takes an array argument, that way we can make an override for subqueries and both will read well:
```kt
fun <T> anyFrom(array: Array<T>): AnyFromArrayOp<T> = AnyFromArrayOp(array)

// use
WHERE { Users.id eq anyFrom(arrayOf(1, 2)) }
WHERE { Users.id eq anyFrom(Users.selectAll()) }
```

Also, please place the functions in the subsection titled "// Array Comparisons".